lib/diagnostic.md

```diff
@@ -2,51 +2,50 @@
 
 Using `inspect element`, write four selectors found on the main page for this GitHub repo.
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
-
+```$("#ajax-error-message").show(function()```
+```t=$.ajax({url:$(this).attr("data-url"),data:n})```
+```$.hashChange```
+```$(document).on ```
```

Comment:
You didn't have to attach methods, but good just to clarify this is a selector: `ajax-error-message`